docs/Deploying/Load-testing.md

```diff
@@ -14,33 +14,35 @@ A test is deemed successful when the Fleet server is able to receive and make re
 
 ## Results
 
-### 1,000 hosts
+### 2,500 hosts
 
 With the following infrastructure, 1,000 hosts successfully communicate with Fleet. The Fleet server is able to run live queries against all hosts.
 
 |Fleet instances| CPU Units       |RAM             |
 |-------|-------------------------|----------------|
-| 1 Fargate task | 256 CPU Units  |512 MB of memory|
+| 1 Fargate task | 512 CPU Units  | 4GB of memory |
 
 |&#8203;| Version                 |Instance type |
 |-------|-------------------------|--------------|
-| Redis | 5.0.6                   |cache.m5.large|
-| MySQL | 5.7.mysql_aurora.2.10.0 | db.t4g.medium|
+| Redis | 5.0.6                   | cache.t4g.medium |
+| MySQL | 5.7.mysql_aurora.2.10.0 | db.t4g.small |
 
 ### 150,000 hosts
 
 With the infrastructure listed below, 150,000 hosts successfully communicate with Fleet. The Fleet server is able to run live queries against all hosts.
 
 |Fleet instance | CPU Units       |RAM             |
 |-------|-------------------------|----------------|
-| 25 Fargate tasks | 1024 CPU units  |2048 MB of memory|
+| 20 Fargate tasks | 1024 CPU units  | 4GB of memory |
```

Comment:
Their boundedness was found by pinning a task size that worked, and then adjusting the number of tasks until it failed. So in this case there are a range of task sizes that work, from many small tasks to a few big tasks but their total memory/cpu when totaled should be about equal (with some variation of course).

Reply:
I can also tell its probably memory boundedness since I change the number of tasks and cpu usage still has plenty free while memory gets somewhat constrained, and because of how memory sampling works even if 50% is "available", the tasks might be running out of memory in reality.